### DISCUSSION TITLE
利用 Python 連結 ChatGPT

### DISCUSSION THREAD

**Discussion by rumy127** [1101-GS4719-C/Python-Community-GS4719-C#1622] [General]
因為要開始著手專題的緣故，接觸了很多跟AI、VR/AR/MR眼鏡相關的資訊，想將AI應用在硬體上(VR,AR眼鏡等)。好比說：能互動對話的虛擬寵物、說出某個地名就能將景象顯示在眼前等等。
最近剛好在尋找能夠把AI API運用到自己專題內容的方法，而現在市面其實已經有非常多種不同公司所創造的AI bot，也有很多open source可供使用。
這邊想分享一篇運用Python來使用OpenAI API的文章，相對其他語言要來得容易理解。雖然他需要付費才能使用，但感覺若能充分運用或許能做出很有趣的作品！
https://aruva.medium.com/building-an-ai-powered-application-with-openai-apis-combining-chatgpt-and-dall-e-in-python-35c5fe7268bd

**Comment by 1chooo** [comment]
雖然是說用到 `openai-api` 需要付費，不過其實每個 `token` 所花的費用不會太高，基本上正常聊天對開發的角度而言是可以接受的，畢竟開發產品的目標就是要賺錢，以公司角度而言，呼叫 `openai-api` 省去了訓練大型語言模型的成本，唯一顧慮的就是 openai 會回傳資料。專題加油啦！

- Open AI 收費方案: https://openai.com/pricing
